go/escaper/escaper.go

```diff
@@ -0,0 +1,48 @@
+package escaper
```

Comment:
Can you rename this package to something like `terminal_escaper`? There are lot of things we could potentially be escaping.

Reply:
I am going with `terminalescaper`, as underscore is not recommended in go package names.

Reply:
Maybe `terminal-escaper`?

Reply:
sure, that's good with me

Reply:
I cannot use the `-` in package names, so I am going with `terminalescaper`

Reply:
ok, that's fine. thanks